microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -496,6 +497,23 @@ def _configure_vdpa_devices(self, node: Node) -> None:
             node.tools[Chown].change_owner(file=PurePath(device_path), user=user)
             node.tools[Chmod].chmod(path=device_path, permission=permission, sudo=True)
 
+    def _set_data_disk(self) -> None:
+        datadisk_name = ""
+        disks = self.node.tools[Lsblk].get_disks()
+        # get the first unmounted disk (data disk)
+        for disk in disks:
+            if disk.is_mounted:
+                continue
+            if disk.name.startswith("sd"):
+                datadisk_name = disk.device_name
+                break
+        # running lsblk once again, just for human readable logs
+        self.node.execute("lsblk", shell=True)
```

Comment:
can lsblk tool be used here?

Reply:
`self.node.tools[Lsblk].get_disks()` gives a json output. While simple `lsblk` command output is more readable. I'll change it to `lsblk.run()`